app/routes/_extras.blog.rss[.xml].tsx

```diff
@@ -0,0 +1,46 @@
+import type { LoaderFunction } from "@remix-run/node";
+import { getBlogPostListings } from "~/lib/blog.server";
+
+const getCData = (text: string) => {
+  return `<![CDATA[${text}]]>`;
+};
+
+export const loader: LoaderFunction = async () => {
+  const blogUrl = `https://remix.run/blog`;
+  const posts = await getBlogPostListings();
+
+  const rss = `
+    <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+      <channel>
+        <title>Remix Blog</title>
+        <link>${blogUrl}</link>
+        <atom:link href="${blogUrl}/rss.xml" rel="self" type="application/rss+xml" />
+        <description>Thoughts about building excellent user experiences with Remix.</description>
+        <language>en-us</language>
+        <generator>https://github.com/remix-run/remix</generator>
+        <copyright>© Shopify, Inc.</copyright>
+        ${posts
+          .map(post =>
+            `
+            <item>
+              <title>${getCData(post.title)}</title>
+              <description>RFC-822 date: ${getCData(post.summary)}</description>
```

Comment:
I'm assuming this just got left behind from wherever you took this code?

```suggestion
              <description>${getCData(post.summary)}</description>
```

Reply:
Thank you very much for pointing that out @brookslybrand!

I used the [W3C Feed Validation Service](https://validator.w3.org/feed/check.cgi) to verify if the RSS feed generated was correct and encountered an error as follows.

Initially, I set `<pubDate>` using `toISOString()`, which led to this error. 

![スクリーンショット 2024-02-08 10 12 33](https://github.com/remix-run/remix-website/assets/66447334/5808b0ea-6cc4-41b4-8081-6b798557ec5a)

In the process of resolving this error, I experimented with various date and time formats to see which would not cause an error. Somehow, and I don't know why, I got confused between `<description>` and `<pubDate>` and kept trying through trial and error.

So, this part was simply something I forgot to remove from when I fixed that error.! I'm truly sorry for the inconvenience caused. 🙇

see: https://github.com/remix-run/remix-website/pull/184/commits/0d217880ccd8ae66c62bb9b34e194e026f40328e

Reply:
Haha no worries, I figured it was something like that. Thanks so much for validating the output